e2e/addon/mock/testrepo/helm-repo/index.yaml

```diff
@@ -10,7 +10,8 @@ entries:
         - http://127.0.0.1:9098/helm/fluxcd-test-version-1.0.0.tgz
       version: 1.0.0
       annotations:
-        system: "vela>=1.3.0; kubernetes>=1.10.0"
+        system.vela: ">=1.3.0"
```

Comment:
This is mocked Chart repo and index file. Do the two fields work for official Chartmesum or Harbor

Reply:
Yes, have tested. Cause `annotation` can be set with any key-value pair